### PR TITLE
Make cartridge container available to override

### PR DESCRIPTION
**Pull request by ArtDu**
It's needed to be able to reuse TarantoolCartridgeContainer as parent class

I haven't forgotten about:
- [x] Tests
- [x] Changelog
- [ ] Documentation
- [x] Commit messages comply with the [guideline](https://www.tarantool.io/en/doc/latest/dev_guide/developer_guidelines/#how-to-write-a-commit-message)
- [x] Cleanup the code for review. See [checklist](https://github.com/tarantool/cartridge-java/blob/master/docs/review-checklist.md)

Related issues:

### CODE REVIEW

**Review by dkasimovskiy — APPROVED** [review]

File: src/main/java/org/testcontainers/containers/TarantoolCartridgeContainer.java [hunk 1]
```diff
@@ -258,15 +258,15 @@ private static Map<String, String> mergeBuildArguments(Map<String, String> build
...
+    protected static ImageFromDockerfile buildImage(String dockerFile, String buildImageName,
                                                   final Map<String, String> buildArgs) {
         ImageFromDockerfile image;
         if (buildImageName != null && !buildImageName.isEmpty()) {
             image = new ImageFromDockerfile(buildImageName, false);
         } else {
             image = new ImageFromDockerfile();
         }
-        return image.withFileFromClasspath("Dockerfile", dockerFile)
+        return image.withBuildArgs(buildArgs).withFileFromClasspath("Dockerfile", dockerFile)
```
Comment (dkasimovskiy) [inline src/main/java/org/testcontainers/containers/TarantoolCartridgeContainer.java:269]:
```suggestion
        return image.withFileFromClasspath("Dockerfile", dockerFile)
```

**Review by dkasimovskiy — CHANGES_REQUESTED** [review]

File: src/main/java/org/testcontainers/containers/TarantoolCartridgeContainer.java [hunk 2]
```diff
@@ -85,45 +85,45 @@
...
+    protected static final int ROUTER_PORT = 3301;
+    protected static final String CARTRIDGE_USERNAME = "admin";
+    protected static final String CARTRIDGE_PASSWORD = "testapp-cluster-cookie";
+    protected static final String DOCKERFILE = "Dockerfile";
+    protected static final int API_PORT = 8081;
+    protected static final String VSHARD_BOOTSTRAP_COMMAND = "return require('cartridge').admin_bootstrap_vshard()";
+    protected static final String SCRIPT_RESOURCE_DIRECTORY = "";
+    protected static final String INSTANCE_DIR = "/app";
+
+    protected static final String ENV_TARANTOOL_VERSION = "TARANTOOL_VERSION";
```
Comment (nickkkccc) [inline src/main/java/org/testcontainers/containers/TarantoolCartridgeContainer.java:98]: Can we make public constants with the ENV_ prefix for ease of use when you add them as Dockerfile build args?

Comment (ArtDu) [reply src/main/java/org/testcontainers/containers/TarantoolCartridgeContainer.java]: Good idea

Revealed them

File: src/main/java/org/testcontainers/containers/TarantoolCartridgeContainer.java (hunk 1 above)
Comment (ArtDu) [reply src/main/java/org/testcontainers/containers/TarantoolCartridgeContainer.java]: Fixed it

**Review by dkasimovskiy — APPROVED** [review]

**Review by nickkkccc — APPROVED** [review]

**Pull request merged**